### DISCUSSION TITLE
Golang not working in VSCode

### DISCUSSION THREAD

**Discussion by samchouse** [jdx/mise#333] [Q&A] [answered]
When starting VSCode, I get this error. I'm using zsh.

![image](https://user-images.githubusercontent.com/46873232/225465377-4ee75c3b-ed4d-49c3-94a6-c14384de8f6f.png)

**Comment by jdx** [comment, marked as answer]
perhaps try adding it to your global config with `rtx global golang@latest`

**Reply by samchouse** [reply]
I have, I also have it as my local config. Could it be that VSCode doesn't initialize using `.zshrc`?

**Reply by jdx** [reply]
maybe try ~/.zprofile or ~/.profile—or use shims
